tests/main/ubuntu-core-create-user/task.yaml

```diff
@@ -7,6 +7,15 @@ restore: |
     sed -i '/^mvo/d' /var/lib/extrausers/group
     rm -rf /home/mvo
 execute: |
+    if [ "$MANAGED_DEVICE" = "true" ]; then
+        if snap create-user --sudoer mvo@ubuntu.com 2>create.error; then
+            echo "Did not get expected error creating user in managed device"
+            exit 1
+        fi
+        grep "cannot create user: device already managed" create.error
+        rm create.error
```

Comment:
Why isn't this using the usual pattern we have elsewhere of saying output=$(...)?

The file should probably be removed on restore, if it's necessary.